draft-ietf-httpbis-cache-latest.xml

```diff
@@ -531,6 +531,15 @@
    Also, note that unsafe requests might invalidate already-stored responses;
    see <xref target="invalidation" />.
 </t>
+<t>
+   A response that is stored or storable can be used to satisfy multiple
+   requests, provided that it is allowed to reuse that response for the requests
+   in question. This enables caches to "collapse" multiple requests into a single forward
```

Comment:
add a parenthetical more fully describing what's happening

Reply:
Looking at this again, "multiple requests into a single forward request" says all that needs to be, I think. I will change "requests" to "incoming requests" to clarify.

Reply:
```suggestion
   in question. This enables caches to "collapse" multiple incoming requests into a single forward
```